examples/test_helper.rs

```diff
@@ -12,7 +12,6 @@ fn test_helper_version(ctx: &Context, _args: Vec<RedisString>) -> RedisResult {
     Ok(response.into())
 }
 
-#[cfg(feature = "test")]
```

Comment:
This is for unit testing (to test internal implementation) - seems the feature `test` is still needed for this one?

Reply:
Oh right, this is an example and anyway not going to be part of the crate binary.
Thanks.